.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,100 @@
+name: Bug Report
+description: Report a problem
+labels: ["needs/triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to submit a bug report.
+        Please fill the template below so that we can investigate what's going wrong and hopefully fix it.
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Describe the problem in one or two sentences.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Provide detailed steps our engineers can follow to reproduce the problem.
+      placeholder: |
+        1.
+        2.
+        3.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current Result
+      description: Explain what currently happens after following those steps.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Result
+      description: Explain what you think should happen instead.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Relevant Logs
+      description: Copy and paste any relevant logs (don't screenshot your logs). This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: input
+    attributes:
+      label: URL
+      description: If this problem occurs on a specific URL, please provide it here.
+      placeholder: https://eu1.cloud.thethings.network/console/...
+  - type: dropdown
+    attributes:
+      label: Deployment
+      description: What deployment are you using?
+      options:
+        - The Things Stack Enterprise (self-hosted)
+        - The Things Stack Cloud
+        - The Things Stack Community Edition
+        - The Things Stack Open Source (self-hosted)
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: The Things Stack Version
+      description: Provide the version of The Things Stack that you're using (run `ttn-lw-stack version` / `ttn-lw-cli version`, or see the bottom right in the Console).
+      placeholder: 3.x.x
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: What operating system are you using?
+      options:
+        - Linux (64-bit Intel)
+        - Linux (other)
+        - Windows (64-bit Intel)
+        - Windows (other)
+        - MacOS (64-bit Intel)
+        - MacOS (Apple Silicon)
+        - Other
+    validations:
+      required: true
```

Comment:
Not sure what the relevance is of this.

I would indeed ask for what the client is "using", but it'll depend on the context what is relevant: browser (when using Console), OS (when using TTSOS, TTSE or CLI).

Since many issues are filed by end users, @kschiffer @ryaplots what kind of information do you need here?

Reply:
it looks really nice from my side, and agree with Johan maybe we can just mention the usage and people can mention the OS or browser or the thing that may affect situation.

Reply:
Browser (name and version) would be nice to know indeed. I agree with Johan that bugs can be reported from two perspectives: user and operator, which makes the template ambiguous. Maybe we want to split that up into two separate templates in the future.